cmd/lncli/cmd_pay.go

```diff
@@ -459,7 +455,8 @@ func trackPayment(ctx *cli.Context) error {
 	}
 
 	req := &routerrpc.TrackPaymentRequest{
-		PaymentHash: hash,
+		PaymentHash:     hash,
+		InflightUpdates: true,
```

Comment:
Isn't this supposed to be `NoInflightUpdates` ?

Reply:
It says comment is outdated. In the current revision, it uses `NoInflightUpdates: false` (implicitly)

Reply:
Ok, turned out to be a non-building earlier commit. Fixed now